doc/user/content/integrations/cli/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Materialize CLI
+description: The Materialize command-line interface (CLI).
+menu:
+  main:
+    parent: integrations
+    name: CLI
+    identifier: cli
+    weight: 6
+disable_list: true
+---
+
+`mz`, the Materialize command-line interface (CLI), lets you interact with
+Materialize from your terminal.
+
+You can use `mz` to:
+
+  * Enable new regions
+  * Run SQL commands against a region
+  * Create app passwords
+  * Securely manage secrets
+  * Invite new users to your organization
+
+## Getting started
+
+1. Install `mz`:
+
+   ```shell
+   # On macOS:
+   $ brew install materialize/materialize/mz
+   # On Ubuntu/Debian:
+   $ apt install mz
+   ```
+
+   See [Installation](installation) for additional installation options.
+
+2. Log in to your Materialize account:
+
+   ```shell
+   $ mz auth login
+   ```
+
+   `mz` will launch your web browser and ask you to log in.
+
+   See [Configuration](configuration) for alternative installation methods.
+
+3. Show enabled regions in your organization:
+
+   ```shell
+   $ mz region list
+   aws/us-east-1  enabled
+   aws/eu-west-1  disabled
+   ```
+
+4. Launch a SQL shell connected to one of the enabled regions in your
+   organization:
+
+   ```shell
+   $ mz sql aws/us-east-1
+   psql (14.2)
+   Type "help" for help.
```

Comment:
since we don't have control over psql prompts after connecting to the shell, it would be great to print some details on connection like:

```
authenticated using <profile> profile
connected to <cluster> cluster in <region>
psql (14.2)
```